oex_plugin/openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -24,7 +25,18 @@ def retrieve(original_func, self, request, username):
     english_proficiency_options = [
         {'value': eng_prof.code, 'label': eng_prof.label} for eng_prof in EnglishProficiency.objects.all()
     ]
-    account_settings[0]['english_proficiency_options'] = english_proficiency_options
+    total_employee_options = TotalEmployee.objects.get_choices()
+    org_type_options = OrgSector.objects.get_choices()
```

Comment:
I would cache those values in the onboarding app if it is not already cached.

Is it?

Reply:
The goal was simply to port the existing implementation, not improve it. That's why I didn't do it.

Reply:
ok, let it be

I'll note that in the TechDebt